### PR TITLE
(LTH-62) Publish tagged artifacts to github releases

### PR DESCRIPTION
**Pull request by branan**

### CODE REVIEW

**Comment by branan** [comment]
We need to generate a token for this, still. Other than that it's good to go

File: appveyor.yml [hunk 1]
```diff
@@ -16,8 +16,23 @@ install:
...
 
 test_script:
   - ps: ctest -V 2>&1 | %{ if ($_ -is [System.Management.Automation.ErrorRecord]) { $_ | c++filt } else { $_ } }
+
+artifacts:
+  - path: leatherman.7z
+    name: leatherman.7z
+
+deploy:
+  description: Leatherman bulid from AppVeyor
```
Comment (MikaelSmith) [inline appveyor.yml]: Typo

File: .travis.yml [hunk 2]
```diff
@@ -52,3 +52,13 @@ env:
...
     - TRAVIS_TARGET=RELEASE
     - TRAVIS_TARGET=DEBUG
+deploy:
+  provider: releases
+  api_key:
+    secure: iAY48frWHgrNy2kU59JTeZPUQ5gJNGgm9orZuKQU+PGhmmP1nwj/wVMqiNGBz5cTz9YSr5T5/vf1RlppyyMlhqmrLgf9AoZHPGN07Ehwd3RXuTixl05/MThsidvdWkPqXjnQ3zKmAgcuzBMkrRSchkcLB0KztPUfMow75tL30rw=
+  file: leatherman.tar.gz
+  skip_cleanup: true
+  on:
+    repo: branan/leatherman
```
Comment (MikaelSmith) [inline .travis.yml]: This needs to change?

**Comment by branan** [comment]
@MikaelSmith nits addressed

**Comment by branan** [comment]
@kylog I think for appveyor whoever's account our tests are under needs to use their encryption tool to encrypt the token. (I hope it's per-account, and not encrypted to a single key O_O)

For Travis encryption is per-github-repo, so I can generate the token for that

**Comment by branan** [comment]
@kylog @MikaelSmith I couldn't find the password to our jenkins account anywhere in the confluence void, which leads me to believe it wasn't the best choice for our "shared tokens" repository. I'm going to file a ticket that says we need to sort out what to do with shared tokens there.

For now, this is updated with a new token generated by me, and encrypted for travis and appveyor for the PL accounts. It should be mergeable.

**Comment by MikaelSmith** [comment]
:+1:

**Pull request merged**